test/core/direct-deps-leading-to.test.ts

```diff
@@ -41,13 +41,28 @@ describe('directDepsLeadingTo', () => {
     expect(depGraph.directDepsLeadingTo(pkg)).toEqual(expected);
   });
 
-  test('it works with a cyclic dep-graph', () => {
-    const cyclic = depGraphLib.createFromJSON(
-      helpers.loadFixture('cyclic-dep-graph.json'),
-    );
-    const pkg = { name: 'baz', version: '4' };
-    const expected = [{ name: 'foo', version: '2' }];
-
-    expect(cyclic.directDepsLeadingTo(pkg)).toEqual(expected);
+  describe('get direct deps for cyclic graphs', () => {
+    test('it works with a simple cyclic dep-graph', () => {
+      const cyclic = depGraphLib.createFromJSON(
+        helpers.loadFixture('cyclic-dep-graph.json'),
+      );
+      const pkg = { name: 'baz', version: '4' };
+      const expected = [{ name: 'foo', version: '2' }];
+
+      expect(cyclic.directDepsLeadingTo(pkg)).toEqual(expected);
+    });
+    test('it works with a surreal cyclic dep-graph', () => {
```

Comment:
can you please help my OCD and add a whitespace between tests? 🙇

Reply:
yes yes :D